src/Sarif.Viewer.VisualStudio.Core/Tags/KeyEventTextViewCreationListener.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ComponentModel.Composition;
+using System.Windows.Media;
+
+using Microsoft.Sarif.Viewer.ErrorList;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Classification;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Tagging;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.Sarif.Viewer.Tags
+{
+    /// <summary>
+    /// Establishes an <see cref="IAdornmentLayer"/> to place the adornment on and exports
+    /// the <see cref="IWpfTextViewCreationListener"/> that instantiates the adornment on
+    /// the event of a <see cref="IWpfTextView"/>'s creation.
+    /// </summary>
+    [Export(typeof(IWpfTextViewCreationListener))]
+    [ContentType("code")]
+    [TextViewRole(PredefinedTextViewRoles.Document)]
+    internal sealed class KeyEventTextViewCreationListener : IWpfTextViewCreationListener
+    {
+#pragma warning disable IDE0044 // Assigned by MEF
+#pragma warning disable CS0169 // Variable is never used
+#pragma warning disable CS0649 // Filled in by MEF
+        /// <summary>
+        /// Defines the adornment layer for the adornment. This layer is ordered
+        /// after the selection layer in the Z-order.
+        /// </summary>
+        [Export(typeof(AdornmentLayerDefinition))]
+        [Name(nameof(KeyEventAdornmentManager))]
+        [Order(After = PredefinedAdornmentLayers.Selection, Before = PredefinedAdornmentLayers.Text)]
+        private AdornmentLayerDefinition editorAdornmentLayer;
+
+        [Import]
+        public IViewTagAggregatorFactoryService TagAggregatorFactoryService;
+
+        [Import]
+        private ISarifErrorListEventSelectionService sarifErrorListEventSelectionService;
+#pragma warning restore CS0649 // Filled in by MEF
+#pragma warning restore CS0169 // Variable is never used
+#pragma warning restore IDE0044 // Assigned by MEF
```

Comment:
do we like to use the same order, like by alphabetical

Reply:
updated